contracts/SaleVesting.sol

```diff
@@ -34,15 +33,24 @@ contract SaleVesting is Ownable {
     uint256 public TGEDate;
     uint256 public totalClaimed;
     uint256 public totalVestingAmount = 0;
+    //Maximun number of vesting detail array.
+    uint256 internal immutable maxVestingDetailArray = 20;
```

Comment:
If we're not setting in the contructor, lets make this constant.  Also, lets change the scope to private if this contract is not being inherited.  Linting might complain its not uppercase...

```suggestion
    uint256 private const maxVestingDetailArray = 20;
```